doc/changelog.rst

```diff
@@ -1,6 +1,24 @@
 Changelog
 =========
 
+.. rubric:: 1.4
+
+- Fix estimation of image noise (estimates were about 2.7× too high).
+- Add command-line arguments as HISTORY keywords in the output FITS files.
+- New command-line option ``-i rfi-mask`` for katdal data sets, to skip
+  channels that the observation indicates are known to be affected by RFI.
+- Workaround for a small number of MeerKAT observations that had the
```

Comment:
Mmm, maybe `katdal` should also be aware of this.

Reply:
Possibly. The bug was that the `sdp.` prefix was missing on a few stream types (`sdp.continuum_image` in particular, possibly also `sdp.spectral_image` - I'd need to check). I'm not sure if that would affect L2 applycal.